### ISSUE TITLE
No games inserted in Library

### ISSUE THREAD

**Issue by luxzg** [nosyn00b/playnite-fanatical-plugin#2]
Hi!

First of all thanks for the time you spent writing the plugin :-)

But I have a big issue... Fanatical "library" in Playnite keeps staying empty.

Plugin is authenticated, and it makes the update, no obvious errors in GUI anywhere, but no new games added.

I've checked the Fanatical page, and sure enough I have a bunch of games that aren't in any of the existing libraries still unused.

I ran the update and checked the latest log entries, here it what it says:

extensions.log

`24-04 01:41:31.796|INFO |FanaticalLibrary#FanaticalAccountClient:Current Playnite.SDK, Version=6.2.2.0, Culture=neutral, PublicKeyToken=null does not support Javascript Event Messaging. Using old way unoptimized strategy
24-04 01:41:32.385|INFO |FanaticalLibrary#FanaticalAccountClient:Current authentication token is valid.
24-04 01:42:06.234|INFO |CheckDlc#PluginDatabaseObject 4:Task Refresh() - 00:00.00 for 0/0 items
24-04 01:42:06.301|INFO |HowLongToBeat#PlaynitePlugin 1:Task OnLibraryUpdated() - 00:00.00 for 0/0 items
24-04 01:42:26.866|INFO |FanaticalLibrary#FanaticalAccountClient:Current Playnite.SDK, Version=6.2.2.0, Culture=neutral, PublicKeyToken=null does not support Javascript Event Messaging. Using old way unoptimized strategy
24-04 01:42:27.333|INFO |FanaticalLibrary#FanaticalAccountClient:Current authentication token is valid.
24-04 01:42:35.714|INFO |FanaticalLibrary#FanaticalAccountClient:Current Playnite.SDK, Version=6.2.2.0, Culture=neutral, PublicKeyToken=null does not support Javascript Event Messaging. Using old way unoptimized strategy
24-04 01:42:36.181|INFO |FanaticalLibrary#FanaticalAccountClient:Current authentication token is valid.
24-04 01:42:36.976|DEBUG|FanaticalLibrary#FanaticalLibrary:Found 437 library Fantical items.
24-04 01:42:36.976|DEBUG|FanaticalLibrary#FanaticalLibrary:Skipped 0 items that are non-games.
24-04 01:42:38.109|INFO |CheckDlc#PluginDatabaseObject 4:Task Refresh() - 00:00.00 for 0/0 items
24-04 01:42:38.146|INFO |HowLongToBeat#PlaynitePlugin 1:Task OnLibraryUpdated() - 00:00.00 for 0/0 items`

playnite.00000.log - nothing

I see from log that it found 437 Fanatical items, and skipped none, yet it still updates 0/0 which is weird.

I don't know what else to try. If I can in any way further help debugging please let me know.

Cheers!

**Comment by pho3nixx** [comment]
I can confirm this problem. Funnily, after setting up Playnite with all the extensions for the first time, also Fanatical worked. Some days later, the Fanatical plugin stopped working and since then doesn't show any games. Deleting C:\Users\USERNAME\AppData\Roaming\Playnite\ExtensionsData\ef17cc27-95d4-45e7-bd49-214ba2e5f4b2 didn't help, neither re-authenticating.
```
27-12 12:56:24.240|INFO |FanaticalLibrary#FanaticalAccountClient:Current Playnite.SDK, Version=6.8.0.0, Culture=neutral, PublicKeyToken=null does not support Javascript Event Messaging. Using old way unoptimized strategy
27-12 12:56:24.504|INFO |FanaticalLibrary#FanaticalAccountClient:Current authentication token is valid.
27-12 12:56:24.836|DEBUG|FanaticalLibrary#FanaticalLibrary:Found 22 library Fantical items.
27-12 12:56:24.836|DEBUG|FanaticalLibrary#FanaticalLibrary:Skipped 0 items that are non-games.
```

**Comment by nosyn00b** [comment]
@luxzg do you still have the problem?

**Comment by luxzg** [comment]
Hey there! I have to admit I haven't started update of Playnite library in a while. I did redeem all games in the meantime, but didn't take much notice about Fanatical in a while. I've opened issue a year ago, but I had some time about 6 months ago and went through all un-redeemed keys in all my accounts... Anyway, I'll try to catch some time tomorrow, update Playnite, all plugins, and give it a nice refresh across all libraries and see what happens. I can then test with and without the option and let you know. Maybe I have some game in there that I didn't redeem myself but gifted to someone, to see if it shows up. I'll try to reply here soon! (Btw that PC is no longer my main, so that's why I didn't update database in a while)
